language/ir-testsuite/tests/transaction_fee_distribution/test_txn_fee_distribution_proper_event_address.mvir

```diff
@@ -1,5 +1,5 @@
 // Make sure that the `SentPaymentEvent` for transaction fee distribution
-// shows 0xFEE as the sender.
+// shows 0xFEE__ as the sender.
```

Comment:
?

Reply:
This is saying `0xFEE` is a prefix. But I should reword the comment to make that more clear.